File: gex/Models/Db/MatchPoolEntry.cs
using Dapper.ColumnMapper;
using gex.Code;
using System;

namespace gex.Models.Db {

    [DapperColumnsMapped]
    public class MatchPoolEntry {

        [ColumnMapping("pool_id")]
        public long PoolID { get; set; }

        [ColumnMapping("match_id")]
        public string MatchID { get; set; } = "";

        [ColumnMapping("added_by_id")]
        public long AddedByID { get; set; }

        [ColumnMapping("timestamp")]
        public DateTime Timestamp { get; set; }

    }
}

File: gex/src/model/MatchPoolEntry.ts

export class MatchPoolEntry {
    public poolID: number = 0;
    public matchID: string = "";
    public addedByID: number = 0;
    public timestamp: Date = new Date();

    public static parse(elem: any): MatchPoolEntry {
        return {
            poolID: elem.poolID,
            matchID: elem.matchID,
            addedByID: elem.addedByID,
            timestamp: new Date(elem.timestamp)
        };
    }

}